init-tracing-opentelemetry/Cargo.toml

```diff
@@ -21,14 +21,15 @@ opentelemetry-jaeger = { version = "0.19", features = [
   "rt-tokio",
 ], optional = true }
 opentelemetry-otlp = { version = "0.13", optional = true, features = [
-  "http-proto",
+  "http-proto", "tls", "tls-roots",
```

Comment:
I guess those features could be removed, with the introduction of the "tls" feature.

Reply:
updated.
BTW: cargo did not seem to like intermixing `dep:` for the otlp feature with specifying the specific features for tls on the same dependency of `opentelemtry-otlp`, so I had previously modified the cargo otlp feature to use the `opentelemtry-otlp/http-proto` instead of the `dep:` declaration.

Reply:
I believe this to be resolved.